### ISSUE TITLE
Document what services are available out-of-the-box

### ISSUE THREAD

**Issue by sethladd** [flutter/flutter#1455]
What can I do with Flutter, by default?  (haptic feedback? trigger intent? play a sound? etc)

At minimum, we can provide a high-level service gallery, with a quick overview of what you get when you open the box. If there's a sample, we can link to it. We can also link to the API docs.

**Comment by eseidelGoogle** [comment]
Maybe this is something @mpcomplete could take a whack at?  I think the desired work product here is a brief document which lists available services, including optional ones like firebase and gcm.  The end result is likely going to be a markdown document on https://github.com/flutter/website, so a pull-request there adding a new services.md might make sense?

**Comment by eseidelGoogle** [comment]
I don't think we'll implement an actual services gallery example as part of this request, but we could file a separate issue for such.

**Comment by sethladd** [comment]
Right, this isn't about a gallery per se, just an enumeration of "what can I do?" and "how do I find the docs, if only for API docs?"

I can't find a canonical place to answer these questions today.

Thanks!

**Comment by sethladd** [comment]
This item is blocked on https://github.com/flutter/flutter/issues/2212 (well, not really blocked, but hard to list services without linking to their docs :)

**Comment by sethladd** [comment]
Unfortunately, this isn't going to happen by the code lab. Too many dependencies.

**Comment by Hixie** [comment]
I don't think we need a bug to track this. It'll be handled by addressing feedback we receive and as a natural result of developing documentation for our plugin work over time.
